presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -399,6 +400,18 @@ public synchronized void renameTable(String databaseName, String tableName, Stri
         }
     }
 
+    @Override
+    public void replaceTableParameters(String databaseName, String tableName, Map<String, String> parameters)
+    {
+        requireNonNull(databaseName, "databaseName is null");
+        requireNonNull(tableName, "tableName is null");
+        requireNonNull(parameters, "parameters is null");
+
+        getRequiredTable(databaseName, tableName);
+
+        alterTable(databaseName, tableName, table -> table.withParameters(parameters));
```

Comment:
`org.apache.hadoop.hive.metastore.api.Table` has more fields than `com.facebook.presto.hive.metastore.Table`. Since `alterTable` has `DROP/CREATE` semantics (in other words "overwrite everyting"), when calling `alterTable` on `com.facebook.presto.hive.metastore.Table` we can overwrite some information that Presto doesn't read/write, but some other tools do.